_posts/2020-05-04-introducing-open-sgid.md

```diff
@@ -20,18 +20,18 @@ It seems so long ago now, but it was less than a year ago when we first [hinted
   <h4 style="margin-top:0;margin-bottom:0;padding-top:0"><strong>Username</strong>: agrc <strong>Password</strong>: agrc</h4>
 </div>
 
-The timing could not be more perfect with the sudden increase in teleworking, which is not expected to wane any time soon, especially here in Utah with the implementation of the [Governor’s teleworking initiative](https://governor.utah.gov/2019/07/16/state-of-utah-introduces-teleworking-program-for-employees/). Not only does the Open SGID benefit Utahns, but it also signals the return to a truly publicly available, direct database connection that can be utilized on a multitude of GIS platforms. Furthermore, it initiates the sunsetting of the state network-only SGID10 database, although both databases (i.e., the Open SGID and the SGID10) will run in parallel for some time to allow users to transition and adapt to the Open SGID.
+The timing could not be more perfect with the sudden increase in teleworking, which is not expected to wane any time soon, especially here in Utah with the implementation of the [Governor’s teleworking initiative](https://governor.utah.gov/2019/07/16/state-of-utah-introduces-teleworking-program-for-employees/). Not only does the Open SGID benefit Utahns, but it also signals the return to a truly publicly available, [direct database connection that can be utilized on a multitude of GIS platforms]({% link sgid/open-sgid/index.md %}). Furthermore, it initiates the sunsetting of the state network-only SGID10 database, although both databases (i.e., the Open SGID and the SGID10) will run in parallel for some time to allow users to transition and adapt to the Open SGID.
```

Comment:
**praise**: way to sneak that link in there :)

Reply:
ha!  thanks!